pkg/app/server/service/webservice/service.proto

```diff
@@ -186,11 +186,9 @@ service WebService {
         option (model.rbac).resource = PROJECT;
         option (model.rbac).action = GET;
     }
-    rpc ListProjectRBACRoles(ListProjectRBACRolesRequest) returns (ListProjectRBACRolesResponse) {}
     rpc AddProjectRBACRole(AddProjectRBACRoleRequest) returns (AddProjectRBACRoleResponse) {}
     rpc UpdateProjectRBACRole(UpdateProjectRBACRoleRequest) returns (UpdateProjectRBACRoleResponse) {}
     rpc DeleteProjectRBACRole(DeleteProjectRBACRoleRequest) returns (DeleteProjectRBACRoleResponse) {}
-    rpc ListProjectUserGroups(ListProjectUserGroupsRequest) returns (ListProjectUserGroupsResponses) {}
     rpc AddProjectUserGroup(AddProjectUserGroupRequest) returns (AddProjectUserGroupResponse) {}
     rpc DeleteProjectUserGroup(DeleteProjectUserGroupRequest) returns (DeleteProjectUserGroupResponse) {}
```

Comment:
Why do those kinds of RPC don't have RBAC config?

Reply:
Sorry, do you mean why we don't need those PRC?

Reply:
No, I meant why those RPC (AddProjectRBACRole, ..., DeleteProjectUserGroup)  don't have 

```
    option (model.rbac).resource =;
     option (model.rbac).action =;
```

Reply:
Oh, I'm just thinking to add that one on the next PR!
Thanks for your confirmation.

Reply:
Got it. 👍